libtiledbsoma/src/soma/managed_query.cc

```diff
@@ -90,6 +90,10 @@ void ManagedQuery::select_columns(
     }
 }
 
+void ManagedQuery::set_column(std::shared_ptr<ColumnBuffer> column_buffer) {
+    column_buffer.get()->attach(*query_);
+}
+
```

Comment:
Why do we need this now?  This was previously covered from `ColumnBuffer`.

Reply:
Didn't mean to add that into this PR. I added this method for array writes.